SportsHub/SportsHub.Api/Exceptions/CustomExceptionModels/CustomException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SportsHub.Api.Exceptions.CustomExceptionModels;
+
+public abstract class CustomException: Exception
+{
+    public int StatusCode { get; set; }
+    public string Message { get; set; }
+    public abstract override string ToString();
```

Comment:
why do you need this to be abstract if there are the same implementations everywhere